### PR TITLE
Update sidebar item name for consistency

### PR DESCRIPTION
**Pull request by unprintedch**
#1571 
Update a string for more consistency

### CODE REVIEW

**Comment by kaitohm** [comment]
@jonathanbossenger Could you check this? 🙇‍♂️

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,4 @@
 /wp-content/themes/pub/wporg-learn-2020/css/print.css
 /wp-content/themes/pub/wporg-learn-2020/css/style-rtl.css
 /wp-content/themes/pub/wporg-learn-2020/css/style.css
+package-lock.json
```
Comment (jonathanbossenger) [inline .gitignore]: @unprintedch could I check if this change was intentional, it's generally not something that gets added to the .gitignore file.

Comment (unprintedch) [reply .gitignore]: It is fixed.

**Comment by jonathanbossenger** [comment]
Initial testing looks good.

**Pull request closed without merging**